File: e2e/selenide-junit5-selenium-grid/src/test/java/data/TaskResponse.java
package data;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class TaskResponse {
    private String id;
    private String title;
    private String description;
    private TaskStatus status;
    private TaskPriority priority;
}

File: demo-ios/DemoUITests/Data/TaskResponse.swift
import Foundation

struct TaskResponse: Codable, Equatable {
    let id: String
    let title: String
    let description: String
    let status: TaskStatus
    let priority: TaskPriority
    let createdDate: String
    let updatedDate: String

    func jsonObject() throws -> Any {
        let data = try JSONEncoder().encode(self)
        return try JSONSerialization.jsonObject(with: data)
    }
}
